messages.go

```diff
@@ -76,8 +78,18 @@ type MediaListResponseBody struct {
 }
 
 type MessageListParams struct {
-	Limit  int `url:"limit,omitempty"`
-	Offset int `url:"offset,omitempty"`
+	Limit                     int    `url:"limit,omitempty"`
+	Offset                    int    `url:"offset,omitempty"`
+	PowerpackID               string `url:"powerpack_id,omitempty"`
+	Subaccount                string `url:"subaccount,omitempty"`
+	MessageDirection          string `url:"message_direction,omitempty"`
+	MessageState              string `url:"message_state,omitempty"`
+	ErrorCode                 int    `url:"error_code,omitempty"`
+	MessageTime               string `url:"message_time,omitempty"`
+	MessageTimeGreaterThan    string `url:"message_time__gt,omitempty"`
```

Comment:
as discussed, please check if these fields have to be single underscore

Reply:
its working as expected. No change required.